add-type/main.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "../common/arg.h"
+#include "../common/db.h"
+#include "../common/path.h"
+#include "../common/product.h"
+#include "../common/session.h"
+#include "../common/util.h"
+
+static void
+usage(void)
+{
+	printf("usage: %s [-h] [-d DIR] TYPE\n", argv0);
+}
+
+int
+main(int argc, char **argv)
+{
+	char *path;
+	const char *type;
+
+	path = NULL;
```

Comment:
Assign with declaration?

Reply:
prefer to align all imports then do declaration, happy to discuss though

Reply:
By standard: "Assign at declaration when possible"